README.md

```diff
@@ -716,11 +702,37 @@ _NOTE: Skip these steps if you are deploying without promoting changes._
    and resolve conflicts in necessary. Conflict resolution should only be
    necessary if cherry-picks occured on the target branch.
 
-4. The merge may have affected `README.md`, the file you are looking at right
+3. The merge may have affected `README.md`, the file you are looking at right
    now. Reopen the file now to ensure you are following the updated version.
 
+4. Now you need to create the release notes. (Skip this step if no link to the release
+   notes document can be found either in the #dcp-ops channel on HCA Slack or in
+   the Google Drive folder mentioned in the [DCP release SOP].
+
+   To produce the list of changes for the DCP release notes, first find the
+   previous release tag for the TARGET branch. Then run:
+
+   ```
+   git LAST_RELEASE_TAG..HEAD --format="%C(auto) %h %s" --no-merges
+   ```
+
+   You will also need to add the release tag and commit hash which are generated
+   later in this guide.
+
+5. At this point you should determine whether or not you will need to reindex.
```

Comment:
Did't you say you wanted to integrate the `git diff` command we talked about.

Reply:
Yes.. added.